src/styles/app.css

```diff
@@ -52,11 +52,9 @@
 
 .header-menu {
     display: flex;
-    flex-direction: row;
-    column-gap: 30px;
-    justify-content: flex-end;
-    align-items: center;
-    margin-left: 100px;
+    justifycontent: space-between;
```

Comment:
i still see the same behavior on smaller screens.  This is how it looks

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/7749e679-3b1e-4989-b0c6-a14253af0501)